draft-ietf-quic-transport.md

```diff
@@ -2486,7 +2486,16 @@ A client will always know whether the server has Handshake keys
 whether the client has Handshake keys.  Under these circumstances, a server
 SHOULD send a CONNECTION_CLOSE frame in both Handshake and Initial packets
 to ensure that at least one of them is processable by the client.  These
-packets can be coalesced into a single UDP datagram (see {{packet-coalesce}}).
+packets can be coalesced into a single UDP datagram; see {{packet-coalesce}}.
+
+A CONNECTION_CLOSE frame that is sent in an Initial packet in response to
+unauthenticated information - the content of Initial or Handshake packets
+primarily - might result in denial of service for a legitimate connection.  QUIC
+does not include defensive measures for on-path attacks during the handshake
+(see {{handshake-dos}}). However, at the cost of reducing feedback about errors
+for legitimate peers, some forms of denial of service can be made more difficult
+for an attacker if endpoints discard illegal packets rather than terminating a
```

Comment:
Is it worth re-stating that the client address can't change during the handshake, so Initial and Handshake packets from a different address can be immediately discarded?

Reply:
Good idea, but I'll add that elsewhere.